brainglobe_atlasapi/atlas_generation/atlas_scripts/template_script.py

```diff
@@ -0,0 +1,111 @@
+from brainglobe_atlasapi.atlas_generation.wrapup import wrapup_atlas_from_data
+
+###Metadata
+__version__ = 1  # The version of the atlas in the brainglobe_atlasapi, this is internal, if this is the first time this atlas has been added the value should be 1
+ATLAS_NAME = None  # The expected format is FirstAuthor_SpeciesCommonName, ie; kleven_rat
+CITATION = None  # DOI of the most relevant citable document
+SPECIES = None  # The scientific name of the species, ie; Rattus norvegicus
+ATLAS_LINK = None  # The URL for the data files
+ORIENTATION = None  # The orientation of the atlas, for more information on how to determine this click here: ........
+ROOT_ID = None  # The id of the highest level of the atlas. This is commonly called root or brain. Include some information on what to do if your atlas is not hierarchical
```

Comment:
> Include some information on what to do if your atlas is not hierarchical

Not sure I understand this: is this information we should include to help people running this script?

Reply:
As in people will use this script to package their own atlases. We should explain how to handle particular edge cases they may run into. for example what if the atlas is not hierarchical? Do we then just say treat everything having a  structure_id_path of brain, region? or what if their atlas doesnt have RGB colours assigned to each region, is it then acceptable for them to randomly generate colours? I think the template should either explain these cases or point to a resource that does.